src/office-toolbox.ts

```diff
@@ -16,13 +16,13 @@ import * as util from './util';
 function logRejection(err) {
   // When the error might contain personally identifiable information, only track the generic part.
   if (err instanceof Array && err.length) {
-    util.appInsightsClient.trackException(err[0]);
+    util.appInsightsClient.trackException({exception: new Error(err[0])});
     for (let message of err) {
       console.log(chalk.default.red(message));
     }
   }
   else {
-    util.appInsightsClient.trackException(err[0]);
+    util.appInsightsClient.trackException({exception: new Error(err)});
```

Comment:
err may be an Error object, in that case it should pass `{exception: err}`.

The code could be refactored so that for all of the cases, the single Error object is extracted and then a single call to trackException is made.